homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -703,3 +950,88 @@ async def async_browse_media(
             media_content_id,
             content_filter=lambda item: item.media_content_type.startswith("audio/"),
         )
+
+    async def async_join_players(self, group_members: list[str]) -> None:
+        """Create a Beolink session with defined group members."""
+
+        # Use the touch to join if no entities have been defined
+        if len(group_members) == 0:
+            await self.async_beolink_join()
+            return
+
+        jids = []
+        # Get JID for each group member
+        for group_member in group_members:
+            jid = self._get_beolink_jid(group_member)
+
+            # Invalid entity
+            if jid is None:
+                _LOGGER.warning("Error adding %s to group", group_member)
```

Comment:
Should we raise instead of continuing on, or is that too harsh? What if one of the entity ids is not a B&O media player, should we raise then?

Reply:
Yes I think raising here is the correct choice.